<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft Resource Pack Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="step1Button1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="step1LineEdit1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="step1Label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Path of the resource pack you want to convert:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1CheckBox1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>70</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Is zipped?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2Option2CheckBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is zipped?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step2Option2Button1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="step2Option2LineEdit1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="step2Option2Label1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Path of the existing resource pack:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="step1Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Step 1: Select the resource pack you want converted:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Step 2: Select the structure of the output resource pack:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Option2Title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Option 2: Use an existing resource pack as a template:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Notes1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*The pack you selected in Step 1 will be restructured in the style that you select in Step 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Minecraft Resource Pack Converter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="step2Notes3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*This will generate a .json file that stores the data structure of this pack</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="step2Option2LineEdit2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="step2Option2Label2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Path of the .json file that will be generated:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="step2Option1LineEdit1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="step2Option1Button1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Option1Title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Option 1: Use an existing .json file as a template:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Option1Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select .json file:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="step2Option2Button2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="step3Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Step 3: Generate the output resource pack:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="step3LineEdit1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="step3Button1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="step3Label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Where to save to the converted resource pack .zip file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="step2Notes2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*You can use default templates, or templates you have generated using Option 2 (useful for modded packs)</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>730</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step3Button2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="step3ProgressLabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Progress: Awaiting start</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="step3Button3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>770</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go To Output Pack</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option1RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>216</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="option2RadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
